docs/middleware/builtin/bearer-auth.md

```diff
@@ -18,6 +18,8 @@ import { bearerAuth } from 'hono/bearer-auth'
 
 ## Usage
 
+NOTE: Your `token` must match the regex `/[A-Za-z0-9._~+/-]+=*/`, otherwise a 400 error will be returned. Notably, this regex acommodates both URL-safe Base64- and standard Base64-encoded JWTs. This middleware does not require the bearer token to be a JWT, just that it matches the above regex.
```

Comment:
How about using [GitHub-flavored Alerts](https://vitepress.dev/guide/markdown#github-flavored-alerts)?

![CleanShot 2025-01-15 at 11 46 36@2x](https://github.com/user-attachments/assets/75b1d936-2d88-47e7-8d75-d320ced2a394)

Reply:
Great idea, thanks. I made that change.